### DISCUSSION TITLE
Signal Coverage Maps Using Measurements and Machine Learning (Project 151)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#16] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #151](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Signal%20Coverage%20Maps%20Using%20Measurements%20and%20Machine%20Learning)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by OxygenFunction** [comment]
I am interested in this project, but have difficulty in getting the dataset. Would you please give me a hint about where I can get some data please?

**Reply by jhalbr** [reply]
I'm happy to hear you are interested in this project! I have updated the project description with a couple of links which may help you locate a data set, including a link to a wireless data portal and a link to a particular data set. I also added a link to a paper which describes the data-driven techniques which can be used with the data sets.

**Reply by Mc-Hf** [reply]
> I'm happy to hear you are interested in this project! I have updated the project description with a couple of links which may help you locate a data set, including a link to a wireless data portal and a link to a particular data set. I also added a link to a paper which describes the data-driven techniques which can be used with the data sets.

Hello, you provided a link to a data set, CRAWDED, related to the project, but I found that it belongs to Dartmouth College and I can't download data from it. Could you tell me is there any way to solve this problem, please?

**Comment by OxygenFunction** [comment]
I have some confusion about the fourth suggested step : _Generate coverage maps using your propagation models and compare your results against theoretical propagation models which are available in Communications Toolbox._
Do I need to use the data which is used in the previous steps to generate a theoretical propagation model by MATLAB Communications Toolbox? I mean I am not sure about how to get a theoretical propagation model by MATLAB for results comparation.

**Reply by jhalbr** [reply]
The "theoretical" propagation models mentioned refer to the models which are pre-defined and ship in Communications Toolbox, such as the ray tracing propagation model. The goal is to generate coverage maps using the pre-defined model and compare them to the coverage maps using your model derived from measurements. [This ](https://www.mathworks.com/help/comm/ug/urban-channel-link-analysis-and-visualization-using-ray-tracing.html)example shows how to use the ray tracing propagation model and generate coverage maps in an urban environment.

**Comment by Mc-Hf** [comment]
Hello, you have provided a link to a data set, "CRAWDED", related to the project, but I found that it belongs to Dartmouth College and I can't download data from it. Could you tell me is there any way to solve this problem, please?

**Reply by robertogl** [reply]
Hello Mc-Hf,

We are happy to know you are working on this project!

To download the data you should register [here](https://crawdad.org/~crawdadmin/registration/registration-form.html) first.
Unless there is any other reason why you cannot download the dataset. In that case let us know and we can try to point you to some other dataset.

**Reply by Kofigo375** [reply]
I registered in order to download the dataset but they emailed me this "Hello, the website [crawdad.org](http://crawdad.org/) is no longer in use.". It turns out they have migrated to IEEE-Dataport but I am finding it difficult to locate the particular dataset to be used. I was able to download the sample dataset from "http://www.mysignals.gr/research.php" but it's for GSM and the reference research paper ([City-Wide Signal Strength Maps: Prediction with Random Forests](https://dl.acm.org/doi/fullHtml/10.1145/3308558.3313726).)was talking about LTE.

**Reply by robertogl** [reply]
Hi @Kofigo375,

In their IEEE Dataport [page ](https://ieee-dataport.org/collections/crawdad
) I found some LTE datasets by just searching 'LTE' in the search bar. Let me know if that works for you. Also, GSM would be fine too.

**Comment by Kofigo375** [comment]
Hi, I am Ephraim Osei, a student at Kwame Nkrumah University of Science and Technology and I am interested in working and collaborating on this project with my team member Darko Christabel Bemah

**Comment by Kofigo375** [comment]
that works for me, thank you

**Comment by anandr16910** [comment]
Hi, I am Anand here is this project thread active now. If yes I shall look into materials provided and propose solution if any.

**Reply by jhalbr** [reply]
Yes, this project is still active.

**Reply by anandr16910** [reply]
I have this dataset, please let me know whether this is compatible with this project. This is not from IEEE dataport or crowdad.
[raw_data.csv](https://github.com/user-attachments/files/18266056/raw_data.csv)
[Network_data.xlsx](https://github.com/user-attachments/files/18266053/Network_data.xlsx)

**Reply by jhalbr** [reply]
I see data with locations and received power measurements, so at first glance this looks like it could be usable for the project.

**Comment by anandr16910** [comment]
Hi,
I am Anand Rajgopalan here, I have already submitted my work and got a reply that MathWorks team will get back to me around 10 days back. But after submitting or sharing my repository in that given form, I had been updating my GitHub repository for better results.

Could you please provide feedback if any.

Regards,
Anand Rajgopalan

**Reply by robertogl** [reply]
Hi Anand, 

Aren't you participating in the AI challenge?

**Reply by anandr16910** [reply]
Yes I would like to participate, could you please share more information about this.?

**Reply by robertogl** [reply]
Information is available [here](https://www.mathworks.com/academia/students/competitions/student-challenge/ai-challenge.html). Your project belongs to the AI trend and you submitted a solution within the deadline. Thus, you are already participating in the challenge (unless you tell me you do not want to). However, I cannot provide substantial feedback to improve your solution, but you have time until July 18 to modify and improve your solution.

**Reply by anandr16910** [reply]
Dear Robert,
Thanks a lot for including my project work in your Mathworks AI trend project list. I see my deadline is 18th July. 
If I see any changes or improvements  to my project I shall commit changes in my GitHub repository before the deadline.

**Reply by robertogl** [reply]
I did not include it. It was already there 😃 .

**Reply by anandr16910** [reply]
I had already submitted my work. I would like to know feedback about my GitHub repo.
